specs/altair/beacon-chain.md

```diff
@@ -120,6 +120,7 @@ This patch updates a few configuration values to move penalty parameters closer
 | - | - |
 | `SYNC_COMMITTEE_SIZE` | `uint64(2**9)` (= 512) |
 | `INACTIVITY_SCORE_BIAS` | `uint64(4)` |
+| `INACTIVITY_SCORE_RECOVERY_RATE` | `uint64(16)` |
```

Comment:
Wasn't this supposed to be `5`?

So that it's
* if leak,
    * active_validator -= 1
    * inactive_validator += 4
* if not leak
    * active_validator -= 6
    * inactive_validator -= 1